pom.xml

```diff
@@ -315,11 +322,6 @@
             <version>${junit-version}</version>
             <scope>test</scope>
         </dependency>
-        <dependency>
-            <groupId>org.apache.maven</groupId>
-            <artifactId>maven-model</artifactId>
-            <version>3.3.9</version>
-        </dependency>
```

Comment:
Just curious, why did we get rid of this? Is it unused?

Reply:
yes, its unused apparently